rust/altrios-core/src/consist/locomotive/locomotive_model.rs

```diff
@@ -30,6 +30,58 @@ impl std::string::ToString for LocoType {
     }
 }
 
+#[altrios_api(
+    #[classmethod]
+    fn from_dict(_cls: &PyType, param_dict: HashMap<&str, f64>) -> anyhow::Result<Self> {
```

Comment:
@panne027 , this enables us to instantiate `LocoParams` in python from a dictionary, which is great because it means we'll make it really clear for users what parameters are expected.